CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.1.0
+
+* Update Faraday version to ~> 0.9
```

Comment:
[nit] since the gem itself doesn't provide it, it's not an "update", so IMO it should be something like "depend on faraday ~> 0.9" or "update faraday dependency to ~> 0.9".